salesagility/SuiteCRM
# Hotfix fixing bugs

## Description




hotfix:
Shows details on screen and also log if error occured and "Display stack trace of errors" is switched on.
This PR also contains some fix of php notices and warns.

## Motivation and Context


Blocking issue of debugging: 
Display stack trace of errors -> cause white screen
Related issue was: https://github.com/salesagility/SuiteCRM/issues/5853

## How To Test This


switch on: "Display stack trace of errors" on admin / system settings
or add to config_overrides:
$sugar_config['stack_trace_errors'] = true;
if any php error occurred, it should shows up on the screen and also should written into error log file.


## Types of changes


### Final checklist
